### PR TITLE
Version >0.60 Fix

### PR DESCRIPTION
**Pull request by Windastella**
Also bump the version to 0.0.4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-native-full-screen",
+  "name": "@windastella/react-native-full-screen",
```
Comment (EduardoJM) [inline package.json:2]: What is the reason to change the package name to a pull request? don't make sense.

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Anthonyzou/react-native-full-screen.git"
+    "url": "git+https://github.com/windastella/react-native-full-screen.git"
```
Comment (EduardoJM) [inline package.json:11]: Same to first comment. What is the reason to change the repositories in the package.json? The intentions is to make a new package forked from here?

**Review by EduardoJM — CHANGES_REQUESTED** [review]
This is fantastic and helped in my new project, but, please, consider my comments here and the intentions of the pull request.

Thanks for this.

**Comment by Windastella** [comment]
I'm having issue with AppCenter Build where it cannot find the repo in GitHub causing the build of my app to fail. SO I create a new npm entry to fix this issue.

**Comment by Windastella** [comment]
I'm closing this pull request as it had not been merge and my repo is diverging because of previous issue.

**Comment by EduardoJM** [comment]
I understand. Anyway, thanks for this. This helped in my code 😄

**Pull request closed without merging**